mwdb/web/src/components/Settings/Views/UsersPendingList.js

```diff
@@ -50,7 +40,7 @@ export default function UsersPendingList() {
             viewAlert.setAlert({
                 success: `User ${login} successfully accepted.`,
             });
-            await getUsers();
+            getPendingUsers();
```

Comment:
I would go with `await` to be able to intercept possible error via `try..catch`
```suggestion
            await getPendingUsers();
```